common/components/ui/Modal.jsx

```diff
@@ -18,8 +18,7 @@ type Props = {
       | 'danger'
       | 'link',
     disabled?: boolean,
-    // $FlowFixMe - Why the fuck doesn't this like onClick?
-    onClick: () => void
+    onClick?: () => void
```

Comment:
@wbobeirne - I think you previously had asked for advice here, there were some cases where the onclick prop was not even specified (different than specified with a null value).

In this case you need to mark it as a an [optional property](https://flow.org/en/docs/types/objects/#toc-optional-object-type-properties)